src/app/profile/yourGiving/historicalView/historicalView.component.js

```diff
@@ -34,11 +40,56 @@ class HistoricalView {
       this.loadingGiftsError = true
     })
   }
+
+  isRecent (year, month) {
+    const today = moment()
+    return today.subtract(3, 'months').isBefore(moment(`${year}/${month}/01`, 'YYYY/MM/DD'))
```

Comment:
Do you think something like this is more intuitive? (Double-check that I got the condition right).

```suggestion
    return today.diff(moment(`${year}/${month}/01`, 'YYYY/MM/DD'), 'months') <= 3;
```